samples_project/Assets/SampleViewer/Scripts/SampleSwitcher.cs

```diff
@@ -7,24 +7,49 @@
 using Esri.ArcGISMapsSDK.Components;
 using Esri.HPFramework;
 using System.Collections;
-using System.Collections.Generic;
-using System.IO;
 using UnityEngine;
 using UnityEngine.Rendering;
 using UnityEngine.SceneManagement;
 using UnityEngine.UI;
 
 public class SampleSwitcher : MonoBehaviour
 {
-    public string APIKey = "";
-    public Dropdown PipelineTypeDropdown;
-    public Dropdown SceneDropdown;
+    public GameObject MenuVideo; 
+    private string APIKey;
     public Button ExitButton;
```

Comment:
```suggestion
    [SerializeField] private Button ExitButton;
```

Reply:
So this is actually from the original script and I didn't make any changes on this ExitButton. But as I mentioned from the previous comments above, since the SampleSwitcher script is acting like a GameController, honestly I just don't really see the real benefits or point to keep those properties private.